Barotrauma/BarotraumaShared/SharedSource/Items/Components/ItemContainer.cs

```diff
@@ -396,7 +394,13 @@ public void OnItemRemoved(Item containedItem)
             activeContainedItems.RemoveAll(i => i.Item == containedItem);
             drawableContainedItems.RemoveAll(i => i.Item == containedItem);
             //deactivate if the inventory is empty
-            IsActive = activeContainedItems.Count > 0 || Inventory.AllItems.Any(it => it.body != null);
+            IsActive = Inventory.AllItems.Any();
+            if (!IsActive)
+            {
+                item.SendSignal("0", "contained_conditions");
+                item.SendSignal("0", "contained_condition_percentages");
+                item.SendSignal("0", "contained_items");
+            }
```

Comment:
I think this should be optimized by not forcing the container to stay active if it doesn't have any of these connections.

Reply:
Changed, now the component only stays active if it has at least one of the three connections, or is containing active items or items with a physics body (same as before).